src/main/kotlin/calculator/console/ConsoleOutput.kt

```diff
@@ -0,0 +1,10 @@
+package calculator.console
+
+import calculator.Number
+import calculator.Output
+
+class ConsoleOutput : Output {
+    override fun write(number: Number) {
+        println(number.toString())
```

Comment:
```suggestion
        println(number)
```
`toString()`은 생략 가능합니다 :)